sky/backends/cloud_vm_ray_backend.py

```diff
@@ -3065,7 +3065,10 @@ def _update_after_cluster_provisioned(
             )
             usage_lib.messages.usage.update_final_cluster_status(
                 status_lib.ClusterStatus.UP)
-            auth_config = common_utils.read_yaml(handle.cluster_yaml)['auth']
+            auth_config = backend_utils.ssh_credential_from_yaml(
+                handle.cluster_yaml,
+                ssh_user=handle.ssh_user,
+                docker_user=handle.docker_user)
```

Comment:
Could we test this for other clouds with `image_id` specified with `docker:xxx`, just to make sure changing this will not affect `ssh` for those?

Or, if we have passed in the `ssh_user` and `docker_user` here, should we remove the argument of `handle.docker_user` and `handle.ssh_user` in the `add_cluster` function below?

Reply:
Good point - tested with `pytest tests/test_smoke.py::test_job_queue_with_docker --gcp`.